app-misc/tracker/tracker-3.5.2-r1.ebuild

```diff
@@ -0,0 +1,108 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit bash-completion-r1 flag-o-matic gnome.org gnome2-utils linux-info meson python-any-r1 systemd vala xdg
+
+DESCRIPTION="A tagging metadata database, search tool and indexer"
+HOMEPAGE="https://wiki.gnome.org/Projects/Tracker https://gitlab.gnome.org/GNOME/tracker"
+
+LICENSE="GPL-2+ LGPL-2.1+"
+SLOT="3/0" # libtracker-sparql-3.0 soname version
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="gtk-doc +miners stemmer test vala"
+RESTRICT="!test? ( test )"
+
+PV_SERIES=$(ver_cut 1-2)
+
+RDEPEND="
+	>=dev-libs/glib-2.52:2
+	>=sys-apps/dbus-1.3.2
+	>=dev-libs/gobject-introspection-1.54:=
+	>=dev-libs/icu-4.8.1.2:=
+	>=dev-libs/json-glib-1.4
+	>=net-libs/libsoup-2.99.2:3.0
+	>=dev-libs/libxml2-2.7
+	>=dev-db/sqlite-3.29.0:3
+	stemmer? ( dev-libs/snowball-stemmer:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-util/glib-utils
+	app-text/asciidoc
+	dev-libs/libxslt
+	$(vala_depend)
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+	gtk-doc? ( dev-util/gi-docgen )
+	test? (
+		$(python_gen_any_dep 'dev-python/pygobject[${PYTHON_USEDEP}]')
+		$(python_gen_any_dep 'dev-python/tappy[${PYTHON_USEDEP}]')
+	)
+	${PYTHON_DEPS}
+"
+PDEPEND="miners? ( >=app-misc/tracker-miners-${PV_SERIES} )"
+
+PATCHES=( "${FILESDIR}/${PN}-3.5.2-define-gnu_source-for-gmtime_r.patch" )
```

Comment:
I suggest always formatting `PATCHES=(...)` as

````
PATCHES=(
	"${FILESDIR}/${PN}-3.5.2-define-gnu_source-for-gmtime_r.patch"
)
````

so that the diff is minimal if and when you add an additional patch.